compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -1096,7 +1098,21 @@ static LogicalResult setContractConfig(IREE::GPU::TargetAttr target,
       }
     }
 
-    tileSizes.emplace_back(std::move(workgroupTileSizes)); // Workgroup level.
+    // Split the parallel and reduction tile sizes.
+    if (pipeline == CodeGenPipeline::LLVMGPUTileAndFuse) {
+      SmallVector<int64_t> reductionTileSizes(
+          numParallelLoops + numReductionLoops, 0);
+      reductionTileSizes[numParallelLoops + numReductionLoops - 1] = tileK;
+      tileSizes.emplace_back(std::move(workgroupTileSizes));
+      tileSizes.emplace_back(std::move(reductionTileSizes));
```

Comment:
The tiling passes in LLVMGPUTileAndFuse do not use `iree_codegen.lowering_config`, they use `iree_gpu.lowering_config` which has a dictionary of named tiling levels. We need to use the gpu lowering config, set thread tile sizes, and add operand promotion to the lowering config. An example of how to set up the lowering config like this can be found here: 

https://github.com/iree-org/iree/blob/f02487583ae36e00d1d2fb79de2b8a9fbdc453e7/compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp#L78-L100

For the thread tile sizes, I think it is this (based on my read of LLVMGPUTileAndDistribute):
```
threadTileSizes = workgroupTileSizes;
threadTileSizes[numParallelLoops - 2] = tileX / workgroupSize[0];
threadTileSizes[numParallelLoops - 1] = tileY / workgroupSize[1];
```

with this, we should hopefully see minimal changes to the pipeline tests as well.